lib/benchee/formatters/console.ex

```diff
@@ -58,17 +67,19 @@ defmodule Benchee.Formatters.Console do
 
   """
   @spec format(Suite.t) :: [any]
-  def format(%Suite{statistics: jobs_per_input,
+  def format(%Suite{scenarios: scenarios,
                     configuration: %{formatter_options: %{console: config}}}) do
-    Enum.map(jobs_per_input, fn({input, jobs_stats}) ->
-      [input_header(input) | format_jobs(jobs_stats, config)]
-    end)
+    scenarios
+    |> Enum.group_by(&(&1.input_name))
```

Comment:
this is probably highly personal, but I really dislike the `&` short hand in any language. I think it's not very readable and side steps the problem of naming what you are dealing with, I'd prefer the `fn -> ...` and when this is all on look into getting credo running on CI and configure it with a rule for this if it exists :D